src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -375,6 +383,123 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
                     {showMailCombo ? <VisibilityOffIcon /> : <VisibilityIcon />}
                   </IconButton>
                 </Grid>
+                <Button
+                  variant="contained"
+                  color="primary"
+                  onClick={() => setIsJoinDialogOpen(true)}
+                >
+                  Instructions
+                </Button>
+                <GordonDialogBox
+                  open={isJoinDialogOpen}
+                  title={`Mailbox Instructions`}
+                  closeButtonClicked={onClose}
+                  maxWidth="md"
+                >
+                  <Grid container>
+                    <Typography sx={{ fontSize: '0.8rem' }}>
+                      <Link
+                        className="disable_select"
+                        style={{
+                          borderBottom: '1px solid currentColor',
+                          textDecoration: 'none',
+                          color: gordonColors.primary.blueShades.A700,
+                          fontSize: '1.1rem',
+                          fontWeight: 'bold',
+                        }}
+                        href="https://m.youtube.com/shorts/FxE5PPS94sc"
+                        underline="always"
+                        target="_blank"
+                        rel="noopener"
+                      >
+                        Salsbury Mailbox
+                      </Link>
+                      <Typography sx={{ fontSize: '0.9rem', fontWeight: 'bold' }}>
```

Comment:
```suggestion
                      <Typography sx={{ fontSize: '0.9rem', fontWeight: 'bold' }}>
```

Typography definitely can have its style in css